README.md

```diff
@@ -1,32 +1,31 @@
 # Pelias
 
-> A modular, open-source geocoder built on top of ElasticSearch for fast geocoding.
-
-[![Gitter](https://badges.gitter.im/pelias/pelias.svg)](https://gitter.im/pelias/pelias?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+> A modular, open-source geocoder built on top of Elasticsearch for fast and accurate global search.
 
 ## Announcement: See our [update](https://github.com/pelias/pelias/tree/master/announcements/2018-01-02-pelias-update/) regarding the Mapzen shutdown
 
 ### What's a geocoder do anyway?
 
-Geocoding is the process of transforming input text, such as an address, or a name of a place—to a location on the earth's surface.
+Geocoding is the process of transforming input text, such as an address, or a name of a place—to a location on the Earth's surface.
 
 ![geocode](https://cloud.githubusercontent.com/assets/4246770/16500453/76d6d8cc-3eb9-11e6-85d8-f57894ba7b73.gif)
 
 ### ... and a reverse geocoder, what's that?
 
-Reverse geocoding is the opposite: returning a list of places near a given point.
+Reverse geocoding is the opposite: returning a list of places near a given latitude/longitude point.
 
 ![reverse](https://cloud.githubusercontent.com/assets/4246770/16506005/a2429288-3ed4-11e6-8af0-7ef78824213f.gif)
 
 ### What makes Pelias different from other geocoders?
 
 - It's completely open-source and MIT licensed
 - It's based on open data, so you can run it yourself
+- It's mostly data-agnostic, so if the default open datasets are not enough, you can import your own data
 - You can install it locally and modify it to suit your needs
 - It has an impressive list of features, such as fast autocomplete
```

Comment:
List another feature (since it says "feature**s**"): interpolation, alt-names language support for admin areas

Reply:
Yeah, this whole section could actually probably be re-written to focus better on the important features we have. It's near the top of the readme so very important to get right.